### DISCUSSION TITLE
Error spam suddenly since a few weeks

### DISCUSSION THREAD

**Discussion by freshpr** [OpenIDC/mod_auth_openidc#1280] [Q&A]
I've been having this spam in my error log since a few weeks. I've upgraded to the latest version of both cjose & mod_auth_openidc but the problem remains.
I'm getting around 1GB error log per day and have no idea why. I have 5 identical apache instances running (1 per auth type) and only 1 instance has this issue.

Everything is still working fine as far as I can see

I attached the log in the hope someone could assist on how to get rid of this error.

**Comment by zandbelt** [comment]
the log file seems incomplete? it should have entries that show the actual state cookie value as in:
```apache
[Wed Nov 13 18:31:05.677326 2024] [auth_openidc:debug] [pid 16] src/http.c(129): [client 172.17.0.1:32946] oidc_http_hdr_in_get: Cookie=mod_auth_openidc_state_q3ULymV8upheJ5DsRrkKkTJt6gI=eyJhbGciOiAiZGlyIiwgImVuYyI6ICJBMjU2R0NNIn0..3DjtIbNlEfcg-kzu.TQEtKhWqaTlgANIwo2d9PIYDbJfwRzqN1cFqdodo8LNl18ihy48OizSOePlHA18kPVVHk0rh5n9grV2saA7-qhi-9uYc6oyi9rEpTmXfGhyaxmNvqO0PYxhQVeY0kBplyaj2JK9L9XXpL4JjCdVIQjTHkp8y5e-KdOucgtXR3GSs5stz62ur3mwseceWZCtnV-PEnz7zxUd5bPjW0TIPeI0hX95kCZQ3UszYln9fbEnmOwfQsF9bMjsXiMgC4CD-n4xouqnctmDF66pXidQU.VutcClfe4SJc7WWF2eXXIg
```

**Comment by freshpr** [comment]
edit:
it is resolved. It was some redirect loop
